torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -1229,6 +1225,25 @@ def test_all_reduce_result_cuda(self):
                 self.assertEqual(result, [_build_tensor(src + 1, expected_value)])
             self._barrier()
 
+        def call_dist_op(self, profiling_title, async_op, op, *args, expect_event=True, **kwargs):
+            with torch.autograd.profiler.profile() as prof:
+                work = op(*args, async_op=async_op, **kwargs)
+                if async_op:
+                    work.wait()
+                    work._get_profiling_future().wait()
+
+            def get_event(partial_key):
+                events = [event for event in prof.function_events if partial_key in event.name]
+                return events[0] if len(events) > 0 else None
+
+            recv_event = get_event(profiling_title)
```

Comment:
Nit: Is it always a `recv` event or are there different type of collective comm. calls here? If the latter is true can we have a name such as `comm_event` which might be less confusing?

Reply:
Yes, that is fine. I was mostly asking because I was curious why it was named `recv_event`.